apps/docs/src/app/core/component-docs/table/examples/table-pagination-example.component.html

```diff
@@ -22,7 +22,8 @@
             fd-button
             [fdMenu]="true"
             [fdMenuTrigger]="itemsPerPageMenu"
-            [ngStyle]="{ 'margin-left': (rtl$ | async) ? '20px' : '0', 'margin-right': (rtl$ | async) ? '0' : '20px' }"
+            [style.margin-left]="(rtl$ | async) ? '20px' : '0'"
+            [style.margin-right]="(rtl$ | async) ? '0' : '20px'"
```

Comment:
having 2 `async` pipes is generally not the best practice. Maybe it would be better to have a `style$` observable in .ts and bind the button to it?

Or, for example, we can use declare a variable with `*ngIf` , since `rtl$` should emit instantly
```html
<button
  *ngIf="rtl$ | async as rtl"
   [style.margin-left]="rtl ? '20px' : '0'"
   [style.margin-right]="rtl ? '0' : '20px'"
```